### PR TITLE
Fix name of RabbitMQ CR instance

### PR DESCRIPTION
**Pull request by kajinamit**
Commit 77be8e1f133cb06bd020c297ab8cb03bc4a95254 removed the existing override and now the CR is deployed with a different name defined in the repository.

Let's re-introduce the name override to avoid updating the cluster name we have defined in each sample file.

### CODE REVIEW

**Review by bshephar — APPROVED** [review]
Seems fine to me for resolving the issue described.

**Comment by kajinamit** [comment]
recheck

File: scripts/gen-service-kustomize.sh [hunk 1]
```diff
@@ -83,6 +83,15 @@ EOF
     fi
 done
 
+if [ -n "$CR_NAME" ]; then
+    cat <<EOF >>kustomization.yaml
+  patch: |-
```
Comment (karelyatin) [inline scripts/gen-service-kustomize.sh:88]: patch: line extra

**Review by karelyatin — CHANGES_REQUESTED** [review]

File: scripts/gen-service-kustomize.sh (hunk 1 above)
Comment (kajinamit) [reply scripts/gen-service-kustomize.sh]: Oh, yes. Good catch. I've fixed it

File: scripts/gen-service-kustomize.sh [hunk 2]
```diff
@@ -38,8 +38,6 @@ fi
 IMAGE=${IMAGE:-unused}
 IMAGE_PATH=${IMAGE_PATH:-containerImage}
 
-NAME=${KIND,,}
```
Comment (kajinamit) [inline scripts/gen-service-kustomize.sh:41]: I noticed this is not used, then we can probably use NAME instead of CR_NAME.

**Review by karelyatin — COMMENTED** [review]
/lgtm

**Review by karelyatin — APPROVED** [review]

**Pull request merged**